### PR TITLE
tools/pd-control: add option of disable-namespace-relocation

### PR DESCRIPTION
**Pull request by lerencao**
related PR: https://github.com/pingcap/pd/pull/1201

### CODE REVIEW

File: tools/pd-control.md [hunk 1]
```diff
@@ -279,6 +280,10 @@ config set cluster-version 1.0.8              // 设置 cluster version 为 1.0.
 `disable-location-replacement` 用于关闭隔离级别检查。
 当设置为 true 时，PD 不会通过调度来提升 Region 副本的隔离级别。
 
+`disable-namespace-relocation` 用于关闭 Region 的空间调度。
```
[the remark this answers is not in the document]
Comment (lerencao) [reply tools/pd-control.md]: fixed

File: tools/pd-control.md [hunk 2]
```diff
@@ -279,6 +280,10 @@ config set cluster-version 1.0.8              // 设置 cluster version 为 1.0.
 `disable-location-replacement` 用于关闭隔离级别检查。
 当设置为 true 时，PD 不会通过调度来提升 Region 副本的隔离级别。
 
+`disable-namespace-relocation` 用于关闭 Region 的 namespace 调度。
+当设置为 true 时，PD 不会把 Region 调度到它所属的 Store 上。
```
Comment (lilin90) [inline tools/pd-control.md]: Please put line 283 & 284 in the same line.

Comment (lerencao) [reply tools/pd-control.md]: updated.

File: tools/pd-control.md [hunk 3]
```diff
@@ -279,6 +280,9 @@ config set cluster-version 1.0.8              // 设置 cluster version 为 1.0.
 `disable-location-replacement` 用于关闭隔离级别检查。
 当设置为 true 时，PD 不会通过调度来提升 Region 副本的隔离级别。
 
+`disable-namespace-relocation` 用于关闭 Region 的 namespace 调度。当设置为 true 时，PD 不会把 Region 调度到它所属的 Store 上。
+
+
```
Comment (lilin90) [inline tools/pd-control.md]: Please delete this extra blank line (line 285) to keep the document clean and standard.

**Pull request merged**